docs/cli.md

```diff
@@ -178,51 +306,59 @@ And you can create the new config file with your disired name with `--output` fl
 
   You can start installed plugin like below.
   ```
-  ~$ ./vatz plugin start --plugin <pluginName>
+  ~$ ./vatz plugin start --plugin <pluginName> or vatz plugin start --plugin <pluginName> --args <arguments>
   ```
-  For some plugins that require arguments for binary, you can provide it with `--args` flag. It should be surrounded by quotes like as one string. For example,
-
+  For certain plugins that necessitate arguments for the binary, these can be supplied using the --args flag. 
+  The arguments should be enclosed within quotes as a single string. Details about the arguments are provided by the plugin itself. 
+  For instance, in the case of the recently installed plugin, you can refer to https://github.com/dsrvlabs/vatz-plugin-cosmoshub for more information.
+  
   ```
-  ~$ ./vatz plugin start --plugin cosmos-status --args "--valoperAddr=<YOUR VALOPER ADDRESS>" 
+  ~$ ./vatz plugin start --plugin cosmos-status --args "--valoperAddr=5dsxaisdoifb2b194ajsllba7"
```

Comment:
@xellos00 
In other examples, you've included something like <YOUR_VALOPER_ADDRESS>, but is there a reason why that's the only part that resembles the actual value?

Reply:
I don't fully understand your point, but I think my answer can help your question if you are asking about why I used some value that is somewhat real alike only in this example. I'd wanted emphasize that every plugin arguments are different and they need to confirm exact arguments and its value from plugin's own info.  That's why I put the real alike address (which's fake address) in the value on example, and also attached sample plugins explanation for more details at line 311 ~ 316(which include plugin git address too) to let reader confirm for themselves.

Reply:
Oh, I missed checking that. I agree with what you said about that part.